views/game/Help - characters include.tw

```diff
@@ -50,12 +50,12 @@
 <</eve>>
 <br/>
 <<octavia>>
-    I am the leader of my tribe. We are in need of wolf pelts to keep us warm during the coming winter.
+    I am the leader of my tribe, The grounders. We are in need of wolf pelts to keep us warm during the coming winter.
 <</octavia>>
 
 <h3>Underground</h3>
 <<dom>>
-    I am the boss of the Underground streets! <br/>
+    I am Dom Soprano, self-appointed mayor of the Underground city! <br/>
```

Comment:
Not Soprano tho. Just leave it as Dom

Reply:
When Vincent talks about the Underground city he mentions Mr. Soprano...

Fixed!